app/models/user.rb

```diff
@@ -45,6 +45,26 @@ def set_address
     self.address = "#{city}, #{country}" if city.present? && country.present?
   end
 
+  def address_format
+    if address.blank?
+      errors.add(:address, "can't be blank")
+    else
+      parts = address.split(',')
+      if parts.length != 2 || parts.any?(&:blank?)
+        errors.add(:address, 'must include a city and country separated by a comma')
+      end
+    end
+  end
```

Comment:
No need to validate this since the `set_address` method already ensures a valid address is built. Users should not be able to modify the address field. Users should only be able to change their city (country is always 'Philippines') since we're building a local application.

Reply:
deleted address-format